docs/ci-integration/ci-integration.md

```diff
@@ -0,0 +1,118 @@
+## Getting Started
+
+### Dependencies
+
+Earthly has two software dependencies: `docker` and `git`. 
+
+ `docker` is used to glean information about the containerization environment, and manage our `earthly-buildkitd` daemon. It is also used to do things like save images locally on your machine after they have been built by Earthly.
+
+Currently, the `earthly-buildkitd` daemon requires running in `--privileged` mode, which means that the `docker` daemon needs to be configured to allow this as well. Rootless configurations are currently unsupported.
+
+`git` is used to help fetch remote targets, and also provides metadata for Earthly during your build.
+
+Because `earthly` will not install these for you, please ensure they are present before proceeding. These tools are very common, so many environments will already have them installed. If they are not, here are some installation instructions that may help:
+
+#### Linux
+
+To install `git`, you can typically use your distributions package manager. [This page](https://git-scm.com/download/linux) has installation instructions for most distributions.
+
+To install `docker`, use the most recent versions [directly from Docker](https://docs.docker.com/engine/install/#server). The versions packaged for many distributions tend to fall behind.
+
+#### macOS
+
+To install `git`, the easiest way is to install the "XCode Command Line Tools". If you open up `Terminal`, and type:
+
+```go
+git --version
+```
+
+Then macOS will prompt you to install these tools. You can also use the `git` provided installer or Homebrew, if you prefer. [Details can be found here](https://git-scm.com/download/mac).
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-mac). Be sure to grab the correct installer depending on your CPU architecture.
+
+#### Windows
+
+To install `git`, use the  [MSI installer](https://gitforwindows.org/). This will provide `git`, and a Bash shell; which may prove more natural for using Earthly. You may also use your package manager of choice.
+
+To install `docker`, [download and install Docker CE](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Both the HyperV and WSL2 backends are supported, but the WSL2 one is very likely to be faster.
+
+### Installation
+
+Once you have ensured that the dependencies are available, you'll need to install `earthly` itself.
+
+#### Bare Metal
+
+This is the simplest method for adding `earthly` to your CI. It will work best on dedicated computers, or in scripted/auto-provisioned build environments. You can follow our [regular installation guide](https://earthly.dev/get-earthly) to add `earthly` if desired.
+
+If you are going to mostly be working from a WSL2 prompt in Windows, you might want to consider following the Linux instructions for installation. This will help prevent any cross-subsystem file transfers and keep your builds fast. Note that the "original" WSL is unsupported.
```

Comment:
I would remove this WSL remark, since we wouldn't expect our users to run a CI in WSL2.

Reply:
Ill move it to our alt-installation guide.